invenio_search/config.py

```diff
@@ -94,3 +94,6 @@
     "980__b": ["collections.secondary"],
     "542__l": ["information_relating_to_copyright_status.copyright_status"],
 }
+
+SEARCH_ELASTICSEARCH_RECORDS_DOC_TYPE = 'record'
```

Comment:
Please rename it to `SEARCH_DOCTYPE_DEFAULT = 'record'` and move it after `SEARCH_INDEX_DEFAULT`. Thanks